app/connectors/CustomsFinancialsApiConnector.scala

```diff
@@ -33,23 +32,23 @@ class CustomsFinancialsApiConnector @Inject()(appConfig: AppConfig,
                                               metricsReporter: MetricsReporterService)(implicit ec: ExecutionContext) {
 
   def isEmailVerified(implicit hc: HeaderCarrier): Future[EmailVerifiedResponse] = {
-    httpClient.get(url"${appConfig.customsFinancialsApi}/subscriptions/subscriptionsdisplay")
+    httpClient.get(url"${appConfig.customsDataStore}/subscriptions/subscriptionsdisplay")
```

Comment:
Could you please create a new class called DataStoreConnector or CustomsDataStoreConnector and move these endpoints in that class cause this is incorrect to call data store endpoints from FinancialsApiConnector? Also move the tests as well. Then call these methods from DataStoreConnector in EmailController.ApiConnector should be used, only to connect to Financials Api endpoints.

Reply:
Initially was considering this approach at first but kept it simple. Thought perhaps if **deleteNotification** method would also need to be called by **customs-data-store** in the future, then we can just rename the files from "CustomsFinancialApi" to "CustomsDataStore" without having separate files. But this works too thanks for the feedback 👍

Reply:
okay, deleteNotification may not be moved as it is specific to CDS financials. We moved the email address retrieval to data store as email retrieval is being used now by other CDS teams and custom data store is the one common place for other CDS teams